packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -601,7 +601,7 @@ export class CdkToolkit {
           anyRollbackable = true;
         }
         const elapsedRollbackTime = new Date().getTime() - startRollbackTime;
-        print('\n✨  Rollback time: %ss\n', formatTime(elapsedRollbackTime));
+        print('\n✨  Rollback time: %ss\n', formatTime(elapsedRollbackTime).toString());
```

Comment:
If you're going away from the `%ss`, which i support, we should do that here too

Reply:
Actually people can still do %s. This is actually just a good catch that I don't need to do .toString anymore. It was previously more important.